server/src/reporter.ts

```diff
@@ -15,6 +15,56 @@ const validationMessagesEn: ValidationMessage = {
+  'shouldPassValuesToAllAttributes' : 'Reference {0} not found in {1}',
+  'namedArgumentShouldExist' : 'Reference {0} not found in {1}',
+  'linearizationShouldNotRepeatNamedArguments' : 'Reference {0} is initialized more than once during linearization',
+  'namedArgumentShouldNotAppearMoreThanOnce' : 'Reference {0} is initialized more than once',
+  'shouldInitializeAllAttributes' : 'You must provide initial value to the following references: {0}',
+  'shouldImplementAllMethodsInHierarchy' : 'Inconsistent hierarchy. Methods on mixins without super implementation on hierarchy',
+  'shouldUseSelfAndNotSingletonReference' : 'Don\'t use the name within the object. Use \'self\' instead.',
+  'shouldNotReassignConst' : 'Cannot modify constants',
+  'shouldNotHaveLoopInHierarchy' : 'Infinite loop in hierarchy',
+  'shouldOnlyInheritFromMixin' : 'Mixin can only inherit from another mixin',
+  'shouldNotDefineMoreThanOneSuperclass' : 'Bad Linearization: you cannot define multiple parent classes',
+  'superclassShouldBeLastInLinearization' : 'Bad Linearization: superclass should be last in linearization',
+  'shouldNotUseOverride' : 'Method does not override anything',
+  'possiblyReturningBlock' : 'This method is returning a block, consider removing the \'=\' before curly braces.',
+  'shouldUseOverrideKeyword' : 'Method should be marked as override, since it overrides a superclass method',
+  'shouldMatchSuperclassReturnValue' : 'Superclass method returned value does not match current definition',
+  'getterMethodShouldReturnAValue' : 'Getter should return a value',
+  'methodShouldHaveDifferentSignature' : 'Duplicated method',
+  'shouldNotDuplicateVariables' : 'There is already a variable with this name in the hierarchy',
+  'shouldNotDuplicateFields' : 'There is already a field with this name in the hierarchy',
+  'shouldNotDuplicateLocalVariables' : 'There is already a local variable with this name in the hierarchy',
+  'shouldNotDuplicateGlobalDefinitions' : 'There is already a definition with this name in the hierarchy',
+  'shouldNotDuplicateVariablesInLinearization' : 'There are attributes with the same name in the hierarchy: [{0}]',
+  'shouldNotDuplicateEntities' : 'This name is already defined (imported from {0})',
+  'shouldNotImportSameFile' : 'Cannot import same file',
+  'shouldNotImportMoreThanOnce' : 'This file is already imported',
+  'parameterShouldNotDuplicateExistingVariable' : 'Duplicated Name',
+  'methodShouldExist' : 'Method does not exist or invalid number of arguments',
+  'shouldImplementAbstractMethods' : 'You must implement all inherited abstract methods',
+  'shouldNotUseVoidMethodAsValue' : 'Message send "{0}" produces no value (missing return in method?)',
+  'shouldInitializeGlobalReference' : 'Reference is never initialized',
+  'shouldNotDefineUnusedVariables' : 'Unused variable',
+  'shouldNotDefineGlobalMutableVariables' : 'Global variables are not allowed',
```

Comment:
@fdodino En español tras tu corrección quedó: "Solo se permiten las variables globales de tipo const". Por lo tanto, en ingles ¿debería entonces quedar algo así?(Mi ingles escrito no es bueno):
```suggestion
  'shouldNotDefineGlobalMutableVariables' : 'Only are allowed global variables of type const',
```